samples/nrf9160/modem_shell/src/link/link_shell_pdn.c

```diff
@@ -47,6 +48,9 @@ void link_pdn_event_handler(uint8_t cid, enum pdn_event event, int reason)
 	case PDN_EVENT_CNEC_ESM:
 		mosh_print("PDN event: PDP context %d, %s", cid, pdn_esm_strerror(reason));
 		break;
+	case PDN_EVENT_CNEC_EMM:
+		mosh_print("PDN event: PDP context %d, %s", cid, pdn_emm_strerror(reason));
+		break;
```

Comment:
I'd like to see error code and the information about ESM vs EMM in the output. Something like (not sure if it compiles):
```suggestion
	case PDN_EVENT_CNEC_ESM:
		mosh_print("PDN event: PDP context %d, ESM error %d: %s", cid, reason, pdn_esm_strerror(reason));
		break;
	case PDN_EVENT_CNEC_EMM:
		mosh_print("PDN event: PDP context %d, EMM error %d: %s", cid, reason, pdn_emm_strerror(reason));
		break;
```